_vendor/github.com/chef/cookstyle/docs-chef-io/data/cookstyle/cops_chef_correctness_blockguardwithonlystring.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Chef/Correctness/BlockGuardWithOnlyString
+department: Chef/Correctness
```

Comment:
The way that these headers render isn't easy to read.

<img width="187" alt="Screen Shot 2021-02-04 at 12 09 12 PM" src="https://user-images.githubusercontent.com/4400151/106949691-24255300-66e2-11eb-9f87-f18b276ad98a.png">

Since all of the cops are Chef cops, let's go with:

```
Cookstyle
|_ All Cookstyle Cops
|
|_ Deprecation Cops
|
|_ Effortless Cops
|
|_ Modernize Cops
|
|_ Redundant Code Cops
|
|_ Sharing Cops
|
|_ Style Cops
```

We can achieve this by breaking the `department` grouping into two parts. In `Chef/Correctness` is "Chef" the author, product, or something else?

Reply:
We need @tas50 feedback on the department naming.  But that comes directly from the cookstyle repo.

Reply:
Ideally, the frontmatter is:

product:
department:
title;

Caveat(s): I don't know what "chef" is in the classification 

I prefer "title" over "name" because it is slightly less ambiguous.

Reply:
"Name" seems to be the convention in Rubocop land. We could make the case that `name` should be `BlockGuardWithOnlyString` (for example) and then the page title could just concatenate the department and name together.

Reply:
Name is from RuboCop land so we want to retain that. It should potentially be cop_name in the yaml

Reply:
Let me make sure that I understand. For RuboCop purposes, you need
- name: product/department/name
- department: department/name

But there's no reason that we can't add
- product: product
- type: department
- cop_name: name

Do the Rubocop fields need to appear first in the list?

Reply:
What is `product`? I don't see where you're getting that.

Reply:
I made it up. The question is where does `chef` come from in the file name. Is it the repo name?

Reply:
We define it in code

Reply:
I think its the organization that's generating the cop. `Chef` as opposed to Rails or something else in the Rubocop ecosystem. 
Thoughts @tas50